brainstormforce/astra
# AST-1895 - PHP Notice - Array to string conversion on widget.php page.

### Description
Error message on the Widgets page

### Screenshots
https://share.bsf.io/p9uOxeKQ

### Types of changes

Bug fix (non-breaking change which fixes an issue) 



### How has this been tested?
- Enable the debug log
- Navigate to the Appearance => Widgets (notice will be displayed)
(On our end not able to replicate it checked on PHP7.4 but on users end it is showing)

### Checklist:
